frame.h

```diff
@@ -73,26 +73,29 @@ constexpr size_t FRAME_SIZE() {
 
 int8_t _radiansToCappedDegrees(float rad) {
   float deg = degrees(rad);
-  if (abs(deg) > 127.0F) {
+  if (fabsf(deg) > 127.0F) {
```

Comment:
I want to pre-empt any questions about this line. Rather than having a single generic absolute value function, C/C++ have a different function for each type:

- `int` -- `abs`
- `long` -- `labs`
- `long long` -- `llabs`
- `float` -- `fabsf`
- `double` -- `fabs`
- `long double` -- `fabsl`

(There is no absolute value function for `short`)

This was using the wrong one before, and I just didn't notice.